specs/monitoring_integration.adoc

```diff
@@ -0,0 +1,263 @@
+= Spec - Introducing a new component monitoring-integration(1st milestone)
+
+A new component called monitoring-Integration is introduced which integrates Grafana
+with tendrl core.
+
+
+== Problem description
+
+The new monitoring-integration component would primarily focus on dynamic
+configuration of grafana, based on the topology changes in etcd. Some of these
```

Comment:
Are we doing any dynamic configuration based on the topology changes in milestone 1?